api-reference/upload-file-api/client-side-file-upload.md

```diff
@@ -131,11 +131,11 @@ The `signature` is HMAC-SHA1 digest of the string `token+expire` using your Imag
 The Private API key should be kept confidential and only stored on your own servers.
 {% endhint %}
 
-If you are using ImageKit.io [client-end SDK](../api-introduction/sdk.md#client-side-sdks) for file upload, it requires an `authenticationEndpoint` endpoint for getting authentication parameters required in the upload API.
+If you are using ImageKit.io [client-end SDK](../api-introduction/sdk.md#client-side-sdks) for file upload, it requires an `authenticator` parameter. It is an asynchronous function that sends a request to your authentication endpoint and returns the required parameters i.e. `signature`, `token`, and `expire`. This asynchronous function uses a backend server to authenticate the request using your [API private key](../../api-reference/api-introduction/api-keys.md#private-key).
```

Comment:
This sentence seems to convey that the function uses private key. While we want to say this is done in their server.

Reply:
done